src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALLS_COUNT = 10;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < BALLS_COUNT; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
+            System.out.println();
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
In situations like this, there is no need to store the object in a local variable if it will be immediately used. Instead, you can pass/use the expression directly without assigning it to a variable